package.json

```diff
@@ -102,6 +102,7 @@
     "lodash.find": "^4.6.0",
     "lodash.includes": "^4.3.0",
     "node-fetch": "^1.6.3",
-    "parse-diff": "^0.4.0"
+    "parse-diff": "^0.4.0",
+    "voca": "^1.2.0"
```

Comment:
I usually prefer to rely on well-handled module instead of writing own (i.e, lodash, etcs) when there isn't huge constraint about sizes / dependencies. Especially voca is battle tested, probably won't need to worry about edge cases of message strings in general. Is there specific reason to avoid? I can try to switch own impl.